hot_osm/templates/components/mobile/mobile_navigation.html

```diff
@@ -0,0 +1,23 @@
+<div x-show="isMenuOpen"
+     @click.outside="isMenuOpen = false"
+     class="font-barlow nav-breakpoint:hidden"
+     id="mobile-menu">
+    <nav class="space-y-1 pb-3 pt-2 font-medium text-base-18"
```

Comment:
Use `gap` - it's cleaner, and doesn't introduce margins on children, rather it adds gaps between children at the flexbox parent level 👍